make_plant_parts_eia.py

```diff
@@ -869,26 +939,45 @@ def dedup_on_category(self, dedup_df, base_cols, category_name, sorter):
         dedup_df = dedup_df.sort_values(category_name)
         return dedup_df.drop_duplicates(subset=base_cols, keep='first')
 
-    def grab_qualifiers(self,
-                        part_df,
-                        record_name,
-                        id_cols,
-                        denorm_table,
-                        denorm_cols
-                        ):
+    def get_qualifiers(self,
+                       part_df,
+                       record_name,
+                       id_cols,
+                       denorm_table,
+                       denorm_cols
+                       ):
         """
-        Grab qualifier records.
-
-        For an individual compiled dataframe for each of the plant parts, we
-        want to add back in some non-data columns (qualifier columns). When
-        qualitative data is consistent for every record in a plant part, we
-        assign these catagoricals. If the records are not consistent, then
-        nothing is added.
+        Get qualifier records.
+
+        For an individual dataframe of one plant part, we typically have
```

Comment:
When you say "one plant part" do you mean all of the records within the entire dataset that have (for example) "plant_prime" as their true granularity? Or would it include all records that have "plant_prime" as the granularity, regardless of whether it's true or false? Or does it refer to only a single particular plant part (with a given set of ID column values?)

Reply:
It is the middle option here. This is very much outside of the true/false granularity labeling.